airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -103,6 +103,9 @@ class S3Hook(AwsBaseHook):
         :class:`~airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
     """
 
+    conn_type = 's3'
+    hook_name = 'S3'
```

Comment:
Where does this S3 come from? is it for `S3Hook`?

Should we have full name or just the prefix -- I have no strong opinion here

Reply:
We might just need some test to see if class is renamed, this `hook_name` is also updated

Reply:
oh ignore, I now understand this better, we are just assigned the Hook name here